src/clr-core/icon-shapes/utils/icon.service-helpers.ts

```diff
@@ -58,19 +58,11 @@ export function legacyAlias(aliases: IconAliasLegacyObject, registry: IconRegist
   }
 }
 
-export function getIcon(shapeName: string, registry: IconRegistrySources): string {
-  return hasIcon(shapeName, registry) ? retrieveIconFromRegistry(shapeName, registry) : getUnknownIcon(registry);
```

Comment:
`hasIcon` was not compatible here for `strictNullCheck` since TS wanted to check that the null/undefined check could be statically enforced at build time instead of a dynamic func. I couldnt find a way around this or a way to change the `hasIcon`  signature to make this guarantee.